ddl/sequence_test.go

```diff
@@ -744,3 +745,27 @@ func (s *testSequenceSuite) TestUnflodSequence(c *C) {
 	// `select nextval(seq), a from t1 union select nextval(seq), a from t2`
 	// The executing order of nextval and lastval is implicit, don't make any assumptions on it.
 }
+
+// before this PR:
+// single insert consume: 50.498672ms
+// after this PR:
+// single insert consume: 33.213615ms
+func (s *testSequenceSuite) TestBenchInsertCacheDefaultExpr(c *C) {
```

Comment:
this test is quite slow， it will cost more than 1 minute. Maybe we only need to run it in bench test instead of unit test?

Reply:
Make sense, thank u every much, addressed.